packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -15,11 +15,7 @@ const useTotalAccountBalance = (accounts: { balance?: number; currency?: string
     if (!accounts.length) return { balance: 0, currency: total_assets_real_currency };
 
     const balance = accounts.reduce((total, account) => {
-        const base_rate = getRate(total_assets_real_currency || '');
-        const rate = getRate(account.currency || total_assets_real_currency || '');
-        const exchange_rate = base_rate / rate;
```

Comment:
is this exchange_rate no used, or this calulation was wrong? Can you please elaborate the reason of changing the calculation

Reply:
Yes please can you elaborate a little 🙏🏻 I'm a bit lost 😅

@shayan-deriv Can you please have a look at these changes 🙏🏻 I remember you checked all these calculations with QA before and everything was correct.

Reply:
@farzin-deriv sure I'm checking them

Reply:
@shontzu-deriv this change is not correct because I've already checked different conditions with appstore team and qa team. could u please elaborate the reason for this change? was there any new condition which the hook didn't cover?
as you can see in the attached image , I created an account on qa10 and it has only one account with 1000 EUR balance. but instead of 1000 it is showing a wrong number: 
![image](https://github.com/binary-com/deriv-app/assets/100833613/db71f615-b075-4e45-a74e-3d949cd6719b)